MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionViewQtGUI.h

```diff
@@ -154,6 +157,19 @@ class MANTIDQT_CUSTOMINTERFACES_DLL EnggDiffractionViewQtGUI
   void setDataVector(std::vector<boost::shared_ptr<QwtData>> &data,
                      bool focused) override;
 
+  void updateFittingDirVec(std::string &bankDir, std::string &focusedFile);
+
+  std::vector<std::string> splitFittingDirectory(Poco::Path selectedfPath);
```

Comment:
@FedeMPouzols as I am using `Poco::Path` here hence it requires but what I can do is pass it as a string instead so Poco import wont be required.